soc/nordic/nrf54h/soc.c

```diff
@@ -43,6 +44,8 @@ sys_snode_t soc_node;
 				      ADDRESS_DOMAIN_Msk |                     \
 				      ADDRESS_BUS_Msk)))
 
+#define dt_nodelabel_cpurad_slot0_partition DT_NODELABEL(cpurad_slot0_partition)
```

Comment:
Names of macros should use uppercase.

Reply:
fixed in https://github.com/zephyrproject-rtos/zephyr/pull/89333